zipkin-lens/src/zipkin/span-node.test.js

```diff
@@ -213,4 +213,39 @@ describe('SpanNodeBuilder', () => {
     expect(root.children.map(n => n.span))
       .toEqual([trace[3], trace[2], trace[1]]); // null first
   });
+
+  it('should order children by timestamp when IPs change ', () => {
```

Comment:
nevermind this.. it is just a backport of another test. Eventhough there is some labor, I wanted to ensure tests are the same across zipkin-ui and zipkin-lens until we complete